presets/4.3/tune/mouseFPV_FFVCycle_Incisor_PT5_tune_filters.txt

```diff
@@ -0,0 +1,151 @@
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: * If tune is too spicy lower the master by a click, or bring in a motor limit
+#$ DESCRIPTION: * Recommended 48k PWM
+#$ DESCRIPTION:
+#$ DESCRIPTION: Options:
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: * **Dynamic Idle:** Enables Dynamic Idle for Freestyle 5"
+#$ DESCRIPTION: * **93% Motor Limit:** Simulates 1800-1900kv motors down closer to 1700kv
+#$ DESCRIPTION: * **Enable Battery Sag Compensation:** Self Explanatory (see tooltip). Land at 3.5v/Cell or it's a bad time.
+#$ DESCRIPTION: * **Heavy Action Camera:** Increases the Pids/Master Slider
+#$ DESCRIPTION: * **Filters -> Include RPM Filter:** Enables bi-directional dshot, sets motor poles to 14, and applies approproiate filter settings. F4 Users should move to 4k pid loop/dshot300
+#$ DESCRIPTION:
+#$ DESCRIPTION: Build Specs This Was Created on:
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: * **Frame:** FPVCycle Incisor/PT5
+#$ DESCRIPTION: * **Motors:** FPVCycle 25mm Emerald 1880kv w/ 93 % Motor Limit 6s
+#$ DESCRIPTION: * **FC:** TMotor F7 (MPU6000)
+#$ DESCRIPTION: * **ESC:** TMotor F55a (non f3)
+#$ DESCRIPTION: * **Action Cam:** GoPro Hero 5 Session
+#$ DESCRIPTION: * **AUW:** 660-670g
+#$ DESCRIPTION:
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/254
+#$ WARNING:  Included Filters set motor poles to 14! Filters Require RPM filtering and DShot telemetry!
+#$ DISCLAIMER: Flash At Your Own Risk!
+#$ FORCE_OPTIONS_REVIEW: TRUE
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# -- End Defaults --
+# -- Begin Mouse Tune --
+
+# -- PID Sliders  --
+set simplified_pids_mode = RPY
+set simplified_d_gain = 135
+set simplified_pi_gain = 100    
+set simplified_feedforward_gain = 70
+set simplified_dmax_gain = 000
+set simplified_i_gain = 090
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 115
+set simplified_master_multiplier = 110
+simplified_tuning apply
+
+# -- iTerm relax --
+set iterm_relax = RP
+set iterm_relax_type = SETPOINT
+set iterm_relax_cutoff = 10
+
+# -- TPA  --
+set tpa_rate = 75
+
+# -- Feedforward --
+set feedforward_boost = 10
+
+# -- Feedforward jitter reduction --
+set feedforward_jitter_factor = 14
+
+# -- Thrust linear  --
+set thrust_linear = 20
+
+# -- DShot Idle --
+set dshot_idle_value = 400
+
+# -- Dyn Idle (off, default) --
+set dyn_idle_min_rpm = 0
+
+# ------ OPTIONS GO BELOW THIS LINE ------
+
+#$ OPTION BEGIN (CHECKED): Dynamic Idle
+set dshot_bidir = ON
+set dshot_idle_value = 150
+set dyn_idle_min_rpm = 20
+set dyn_idle_p_gain = 45
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): 93% Motor Limit (Use this if its too spicy)
+set motor_output_limit = 93
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Enable Battery Sag Compensation?
+set vbat_sag_compensation = 100
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Heavy Action Cam (Hero8/9/10)
+set simplified_master_multiplier = 120
+simplified_tuning apply
+#$ OPTION END
+
+
+#$ OPTION_GROUP BEGIN: Filters (Choose One or None)
+
```

Comment:
i would add two options with dshot 300 and dshot 600 with dshot 300 checked.
ANd add to the working something lile: This tune sets ESC protocol to DSHOT, it is extremely unsafe to even select it, if your ESC does not support DSHOT (can spin the motors without arming)

Reply:
Cool, I broke it out into 2 filter options with 300 and 600 respectively, and added the warning per your suggestion.

Reply:
sorry maybe i wasn't clear. YOu copied the whole filtering section.
Check, for example, how it is done here in this preset:
https://github.com/betaflight/firmware-presets/blob/master/presets/4.3/tune/freedom_spec_460g.txt